client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -268,7 +269,12 @@ const Sidebar = ( { sidebarDomain, siteSlug, submit, goToStep, flow }: SidebarPr
 					{ showPlansModal && site?.ID && (
 						<RecurringPaymentsPlanAddEditModal
 							closeDialog={ () => setShowPlansModal( false ) }
-							product={ { subscribe_as_site_subscriber: true, price: 5 } }
+							product={ {
+								price: 5,
+								subscribe_as_site_subscriber: true,
+								title: translate( 'Paid newsletter' ),
+								type: TYPE_TIER,
```

Comment:
Based on how annual product is created in the modal, looks like I don't need to pass the `type` and title in `annualProduct` prop below this line.

https://github.com/Automattic/wp-calypso/blob/e09c2aaa14b2d84b3e099800237205b58da18aab/client/my-sites/earn/components/add-edit-plan-modal/index.tsx#L226-L232